flang/test/Driver/main_dupes.f90

```diff
@@ -0,0 +1,15 @@
+! UNSUPPORTED: system-windows
+
+! RUN: %clang -o %t.c-object -c %S/Inputs/main_dupes.c
```

Comment:
You can use %cc for the system-compiler and REQUIRE it to be present for this test. 
Reference: https://github.com/llvm/llvm-project/blob/main/flang/test/Runtime/no-cpp-dep.c

Reply:
When I do this, the test goes to "unsupported" for me, as it seems that %cc is not set as a substitution in lit.cfg.py.

Reply:
Check:
* https://github.com/llvm/llvm-project/blob/main/flang/test/lit.site.cfg.py.in#L24, and
* https://github.com/llvm/llvm-project/blob/main/flang/test/lit.cfg.py#L169

Reply:
This change is also gone now and has been removed.